### ISSUE TITLE
[Bug] Aquapazza displays character select portraits in monochrome

### ISSUE THREAD

**Issue by Ninetime** [RPCS3/rpcs3#15468]
### Quick summary

Character portraits appear in black & white during character select.

### Details

In Aquapazza, characters portraits appear monochrome during character select. This appears to be independent of mode.
I've tested with the default configuration and with `write color buffers` set. In both cases, the issue persists.
Of note, this appears to be an Nvidia specific issue. When i run the game with my AMD machine, the colours appear correctly.

### Nvidia
[RPCS3-ninejp-aqpa.log.gz](https://github.com/RPCS3/rpcs3/files/14997208/RPCS3-ninejp-aqpa.log.gz)
![NPUB31336_screenshot_2024_04_16_09_00_09](https://github.com/RPCS3/rpcs3/assets/6248582/8c550894-aa16-42aa-b043-d46c09947ab9)
![NPUB31336_screenshot_2024_04_16_09_00_38](https://github.com/RPCS3/rpcs3/assets/6248582/643a093c-fde7-4fb0-b1fc-3a2347127d1d)

### AMD
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/14997234/RPCS3.log.gz)
![NPUB31336_screenshot_2024_04_16_09_10_03](https://github.com/RPCS3/rpcs3/assets/6248582/63b15c29-69bb-4672-b5e0-aa01760d8be8)
![NPUB31336_screenshot_2024_04_16_09_10_15](https://github.com/RPCS3/rpcs3/assets/6248582/9d0e710a-ebc6-4f93-a52f-8dae4f5ae83b)

### RSX Capture
[NPUB31336_20240416091758_capture.rrc.gz](https://github.com/RPCS3/rpcs3/files/14997363/NPUB31336_20240416091758_capture.rrc.gz)

### Nvidia Machine
AMD Ryzen 7 3700X 
15.92 GiB RAM
NVIDIA GeForce RTX 2060 SUPER (551.23)
Windows 11 22H2

### AMD Machine
AMD Ryzen 7 5800X
31.92 GiB RAM
AMD Radeon RX 6800 XT (24.2.1)
Windows 10 22H2

### Other details

Re-created issue as per @AniLeo's request

**Comment by akisearu2024** [comment]
![Captura de tela 2024-04-16 18 48 25](https://github.com/RPCS3/rpcs3/assets/167251154/2c82cd2d-e532-4a3e-8273-e2df16079a5a)

Fuck off @AniLeo you motherfucker, you want to be censoring my best friend who just wanted to help with your shitty little emulator then you deserve to die brutally, you're portuguese right so you must make gay sex with your dirty ass.

![WhatsApp Image 2020-06-29 at 01 46 54](https://github.com/RPCS3/rpcs3/assets/167251154/103e9169-7b48-4ed3-abfa-37f81fc61394)

**Comment by ToniBC** [comment]
Yes, I have the same problem with an RTX 3070 on the latest versions.

More than anything, to confirm the bug.

Thanks.

**Comment by hardcaz** [comment]
same with rtx 4090

**Comment by digant73** [comment]
@Ninetime, @ToniBC is that still an issue?

**Comment by Ninetime** [comment]
@digant73 Yes, it's still occurring

<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/25c81802-d08a-4726-bf97-12bf7df9779f" />

[RPCS3.log.gz](https://github.com/user-attachments/files/23155767/RPCS3.log.gz)

**Comment by digant73** [comment]
please an update on this issue after many bugfixes on the last months

**Comment by Ninetime** [comment]
> please an update on this issue after many bugfixes on the last months

Still an issue.

[RPCS3.log.gz](https://github.com/user-attachments/files/27923522/RPCS3.log.gz)

[NPUB31336_20260517174252_capture.rrc.gz](https://github.com/user-attachments/files/27923569/NPUB31336_20260517174252_capture.rrc.gz)

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/ad1819dd-6b88-4e7c-8b1b-7d1750f2c3ab" />
